clarify requirements for custom sounds (#4013)
mattermost/docs: source/administration/changelog.md

 - Fixed inconsistencies across product when using ``in:@` / `in:``, such as displaying Direct and Group Messages in ``in:@`` search suggestions.

#### Notifications

>  - Added an option in the **Account Settings** to select different desktop notification sounds. This setting is available in supported browsers and Desktop app since v4.6.0

Reviewer: ```suggestion
 - Added an option in the **Account Settings** to select different desktop notification sounds. This setting is available in supported browsers and in the Desktop app from v4.6.0.
```

>  - Added an option in the **Account Settings** to select different desktop notification sounds. While there are no extra requirements to use custom sounds in the browser, Desktop app users will need at least v4.6.0 to enable the setting.

Reviewer: ```suggestion
 - Added an option in the **Account Settings** to select different desktop notification sounds. This setting is available from v4.6.0 of the Browser and Desktop apps.
```

Author: Hi justine! to try and clarify what I meant: the browser doesn't have any requirements, only the desktop app needs to be 4.6 or greater

Reviewer 2: > This setting is available from v4.6.0 of the Browser and Desktop apps.

You need to split the browser from the Desktop apps. There is no 4.6.0 Browser :)

Perhaps:

This setting is available in the supported Browsers and Desktop apps (v4.6.0+).

Reviewer: Thanks, @Willyfrog and @JtheBAB. Not sure how that one slipped past me, I wasn't focused on the task at hand.
